test/crypto/signing_test.py

```diff
@@ -34,144 +35,151 @@ def test_get_keys_single(self):
     """
     Generating a single key.
     """
-    kg = KeyGenerator(
-      seed = b'TESTSEED9DONTUSEINPRODUCTION99999ZTRFNBTRBSDIHWKOWCFBOQYQTENWL',
-    )
+    with warnings.catch_warnings(record=True) as catched_warnings:
 
-    self.assertListEqual(
-      kg.get_keys(start=0),
+      warnings.simplefilter("always")
```

Comment:
Can this be made more specific, so that it only catches the "Seed has invalid length" warning?

Just in case a regression is introduced in the future that breaks the seed length warning and coincidentally causes an unrelated warning to be emitted (e.g., we add a deprecation warning somewhere, and we forget to update the `Seed` class to account for whatever got changed).